dll/win32/shell32/wine/control.c

```diff
@@ -866,16 +866,116 @@ static	void	Control_DoLaunch(CPanel* panel, HWND hWnd, LPCWSTR wszCmd)
     */
 {
     LPWSTR	buffer;
+#ifndef __REACTOS__
```

Comment:
This function has nothing to do with the original wine code anymore. There is no point in keeping it hacked with all the `#ifdef __REACTOS__` like this. Instead move the original wine version of the function into the large `#ifndef __REACTOS__` block above and the reactos version into the `#else` block.

Reply:
Makes sense. Done.